.github/workflows/verify-fetchy.yml

```diff
@@ -22,6 +22,8 @@ jobs:
           cache: "npm"
       - name: Install dependencies
         run: npm ci
+      - name: Run Tests
+        run: npm test
```

Comment:
Big fan of this! In terms of verification, it will be slightly better coverage if we create a whole new action to run on PR - that will run fetchy & test in parallel, and ensure that we verify fetchy even if tests fail ;)